### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by minaelee**
Multiple minor grammar/punctuation/coherence edits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Then you can use it as `redocly [command] [options]`, for example:
 redocly lint path-to-root-file.yaml
 ```
 
-Minimal required versions of Node.js and NPM are 14.19.0 and 7.0.0 respectively.
+Minimum required versions of Node.js and NPM are 14.19.0 and 7.0.0 respectively.
```
Comment (lornajane) [inline README.md:31]: The minimum required versions of Node.js and NPM are 14.19.0 and 7.0.0 respectively.
```

File: README.md [hunk 2]
```diff
@@ -63,11 +63,11 @@ Your API reference docs are in `redoc-static.html` by default. You can customize
 
 ### Bundle multiple OpenAPI documents
 
-Having one massive OpenAPI description can be annoying, so most people split them up into multiple documents via `$ref`, only to later find out some tools don't support `$ref`, or don't support multiple documents?! Well fear not, because Redocly has a `bundle` command you can use to recombine all of those documents back into one single document. The bundled output Redocly CLI provides is clean, tidy, and looks like a human made it.
+Having one massive OpenAPI description can be annoying, so most people split them up into multiple documents via `$ref`, only to later find out some tools don't support `$ref` or don't support multiple documents. Redocly to the rescue! It has a `bundle` command you can use to recombine all of those documents back into one single document. The bundled output that Redocly CLI provides is clean, tidy, and looks like a human made it.
```
Comment (lornajane) [inline README.md:66]:
```suggestion
Having one massive OpenAPI description can be annoying, so most people split them up into multiple documents via `$ref`, only to later find out some tools don't support `$ref` or don't support multiple documents. Redocly CLI to the rescue! It has a `bundle` command you can use to recombine all of those documents back into one single document. The bundled output that Redocly CLI provides is clean, tidy, and looks like a human made it.
```

File: README.md [hunk 3]
```diff
@@ -79,9 +79,9 @@ redocly lint openapi.yaml
 
 **Configure the rules** as you wish. Other API Linters use complicated identifiers like JSONPath, but Redocly makes life easy with simple expressions that understand the OpenAPI structure. You can either use the [built-in rules](https://redocly.com/docs/cli/rules) to mix-and-match your ideal API guidelines, or break out the tools to build your own.
 
-**Format the output** in whatever way you need, the `stylish` output is as good as it sounds, but if you need JSON or Checkstyle outputs to integrate with other tools, the lint command can output those too.
+**Format the output** in whatever way you need. The `stylish` output is as good as it sounds, but if you need JSON or Checkstyle outputs to integrate with other tools, the `lint` command can output those too.
 
-**Multiple files supported** so you don't need to bundle your API description to lint it, just point Redocly CLI at the "entry point" (e.g.: `openapi.yaml`) and it handles the rest.
+**Multiple files supported** so you don't need to bundle your API description to lint it. Just point Redocly CLI at the "entry point" (e.g.: `openapi.yaml`) and it handles the rest.
```
Comment (lornajane) [inline README.md:84]: This really is all one sentence, so either revert the change or use a semicolon instead.

**Review by lornajane — CHANGES_REQUESTED** [review]
Thanks for the pull request! I added some minor feedback, and I hope that you'll take another look when you have some time.

File: README.md (hunk 3 above)
Comment (minaelee) [reply README.md]: Semicolon it is!

**Review by lornajane — APPROVED** [review]
Thank you!
